src/components/Assets/UploadAssetsComponent.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-unused-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Progress, Icon } from 'semantic-ui-react';
+import Dropzone from 'react-dropzone';
+import '../../_css/UploadAssets.css';
+
+const StatusMessage = props => <div className={`${props.className} `}>{props.message}</div>;
+
+const uploadStatus = (success, error) => {
+  if (success.hasOwnProperty('fail') || error) {
```

Comment:
What is the success format that necessitates the check: `success.hasOwnProperty('fail')`?

Reply:
It is an object with backend response, for instance a partially processed file has below:
{
  fail: "Some assets were skipped. Download the skipped assets file from"
  file: "http://127.0.0.1:8000/api/v1/skipped/"
  success: "Asset import completed successfully Assets that have not been imported have been 
                   written to a file"
}

Reply:
@Ewanjiru Regarding [this comment](https://github.com/AndelaOSP/art-dashboard/pull/177/files#r239504553), can't you talk to Isaiah to update the response to a better format you can work with? I feel like you guys are resorting to "hackery" to make things work, which may come to bite us in the future

Reply:
Noted.